holidayextras/react-date-time-group
# PROD-1397 react-date-time-group does not work with webpack

https://holidayextras.jira.com/browse/PROD-1397

#### What does this PR do? (please provide any background)

This work updates the `prepublish` npm script to add the required plugins and presets so this component can be consumed by a client side project.

#### What tests does this PR have?

No extra tests, tool update only.

#### How can this be tested?

Checkout the **tripapplite** branch of `carHire-demo`.

    npm install
    cd node_modules/tripapp-product-carhire/node_modules/react-date-time-group
    npm install
    npm run prepublish
    cd ../../../..

Then run tripapp as usual, it should build

#### Screenshots / Screencast

Example car hire availability URL:

[http://localhost:5000/?pickUpLocation=&dropOffLocation=&Date=2016-03-01&StartTime=00%3A30&Date=2016-03-01&EndTime=00%3A30&dateOfBirth=#/carhire?from=2016-04-10 12%3A30&to=2016-04-15 13%3A30&date_of_birth=1981-01-01&agent=web1&pickup_location=FL1757&dropoff_location=FL1757](http://localhost:5000/?pickUpLocation=&dropOffLocation=&Date=2016-03-01&StartTime=00%3A30&Date=2016-03-01&EndTime=00%3A30&dateOfBirth=#/carhire?from=2016-04-10 12%3A30&to=2016-04-15 13%3A30&date_of_birth=1981-01-01&agent=web1&pickup_location=FL1757&dropoff_location=FL1757)

Car hire engine URL:

[http://localhost:5000/#/carhire/engine](http://localhost:5000/#/carhire/engine)

#### What gif best describes how you feel about this work?

![giphy](https://cloud.githubusercontent.com/assets/3158640/13856266/456c88a8-ec6c-11e5-8c28-4f3591d8b5ac.gif)

---


#### Reviewers

**Review 1**

**Review 2** \*

**Review 3** _(optional)_

By adding a +1 you are confirming you have...
- Witnessed the work behaving as expected (this could be on the author's machine or screencast).
- Checked for coding anti-patterns.
- Checked for appropriate test coverage.
- Checked all the tests are passing.

\*  for HX this review must be completed by an SE, SA or Project Guru
